superset-frontend/src/dashboard/components/nativeFilters/FilterBar.tsx

```diff
@@ -170,7 +170,19 @@ const StyledFilterControlTitleBox = styled.div`
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  margin-bottom: ${({ theme }) => theme.gridUnit * 2}px;
+  margin-bottom: ${({ theme }) => theme.gridUnit * 1.5}px;
```

Comment:
We might just want to pick 1 or 2 here... I'm _hoping_ we can avoid fractional gridUnits wherever possible.